tests/ui/numbers-arithmetic/shift-near-oflo.rs

```diff
@@ -18,6 +18,7 @@ pub static mut HACK: i32 = 0;
 #[inline(never)]
 fn id<T>(x: T) -> T {
     unsafe { HACK += 1; }
+    //~^ WARN creating a mutable reference to mutable static is discouraged [static_mut_refs]
```

Comment:
Since this test is unrelated, let's fix (or `allow(..)` and mark `FIXME`) this and other similar uses instead.